runtimes/node-14.5/README.md

```diff
@@ -18,22 +18,27 @@ module.exports = async (context) => {
 };
 
 END
+```
+
+2. Generate your `package.json` file:
 
+```bash
+npm init --yes
 ```
 
-2. Build the code:
+3. Build the code:
 
 ```bash
-docker run -e OPEN_RUNTIMES_ENTRYPOINT=index.js --rm --interactive --tty --volume $PWD:/mnt/code openruntimes/node:v3-14.5 sh helpers/build.sh
+docker run --rm --interactive --tty -v $(pwd):/mnt/code:rw -e OPEN_RUNTIMES_ENTRYPOINT=index.js openruntimes/node:v3-14.5 sh helpers/build.sh "npm install"
 ```
 
-3. Spin-up open-runtime:
+4. Spin-up open-runtime:
 
 ```bash
-docker run -p 3000:3000 -e OPEN_RUNTIMES_SECRET=secret-key --rm --interactive --tty --volume $PWD/code.tar.gz:/mnt/code/code.tar.gz:ro openruntimes/node:v3-14.5 sh helpers/start.sh "pm2 start src/server.js --no-daemon"
+docker run -p 3000:3000 -e OPEN_RUNTIMES_SECRET=secret-key --rm --interactive --tty --volume $(pwd)/code.tar.gz:/mnt/code/code.tar.gz:ro openruntimes/node:v3-14.5 sh helpers/start.sh "pm2 start src/server.js --no-daemon"
```

Comment:
Is this change necessary? We do `$PWD` in all other runtime READMEs. If it breaks here, we need to do this change everywhere. If it works without this change, we can revert it from node readmes.

Reply:
It is what's written in the [tests.sh](https://github.com/open-runtimes/open-runtimes/blob/main/tests.sh) file, where I got the working docker commands from. And it's also written like that in the [CONTRIBUTING.md](https://github.com/open-runtimes/open-runtimes/blob/main/CONTRIBUTING.md#installation) file. But I'll change it back to `PWD`. 👍